### ISSUE TITLE
move GF files outside of the repo

### ISSUE THREAD

**Issue by aeantipov** [ALPSCore/hybridization#4]
There are Green's function files, which are used in the code : `types.h` and `green_function.h`. They should be moved to the Green's function repo.

**Comment by egull** [comment]
Yes they should. This is something from which the GF code could start, once Alex is back. Assigning it to Alex as one of the examples of how we use GF.
